### ISSUE TITLE
VIDEO-TDR-FAILURE (nvlddmkm.sys)

### ISSUE THREAD

**Issue by Andr65cmd** [trexminer/T-Rex#831]
Anyone knows how can i fix this ? i am getting this with a blue screen when mining on trex. Setup gigabyte 3070 non lhr directly connected to motherboard 
zotac trinity oc 3080 ti connected on a riser and on a external psu that i am using only for the gpu

**Comment by pdieppa** [comment]
That's an Nvidia driver issue. I would remove and re-install the drivers or try a different version of the drivers.

**Comment by Andr65cmd** [comment]
can you recommend one thats working for you ?because i downgrade the driver but the pc crashed with kernel error 41 meanwhile no error was reported to trex log

**Comment by pdieppa** [comment]
I believe I'm using the latest version 496.49. Have you tried to mine with a single card first?  Try each card individually to confirm both can mine.

**Comment by Andr65cmd** [comment]
i will, but is that a think ? that a gpu wouldnt be able to mine ?

**Comment by pdieppa** [comment]
It could be. The error 41 is not helpful,  is just telling you the computer shutdown improperly. You can specify which card to mine with the mining software.  You need to confirm both cards can mine before moving on to other troubleshooting efforts.

**Comment by Andr65cmd** [comment]
if there is a problem with gpus its definitely the 3080ti.The 3070 was mining for days before i added the 3080ti.But if i test only the 3080ti and the blue screens continue, do i need to send it back for warranty ?

**Comment by pdieppa** [comment]
Well, what I would do is remove the 3070 and install the 3080ti to make sure it can indeed boot up and work normally.  I'm not sure you will receive support because the card does not mine.  The idea is that you want to know 100% the cards are working before making any additional changes to the OS.

**Comment by Andr65cmd** [comment]
Ok yes i agree with you.The pc boots normally with the 3080ti mining nice but the system giving a kernel 41 blue screen randomly also the trex log file didn't record any errors

**Comment by pdieppa** [comment]
Have you made any changes to the virtual memory? I'm not sure exactly what t-rex recommends but I would at least set it to double the RAM on the computer or to the total amount of GPU ram.

**Comment by Andr65cmd** [comment]
yes i have doubled the amount of virtual memory.I have on 750w the main one for the whole pc that now the 3070 is connected and i bought another one 650w that only the 3080ti is connected

**Comment by pdieppa** [comment]
Interesting. Another way to test the card is to load a Linux based mining platform and let it mine using the 3080ti. This should help you decide if there is something going on with the 3080ti. I run HiveOs and boots up from a usb flash drive and includes everything you need to start mining.

**Comment by Andr65cmd** [comment]
ok i will try this also. Also i used lolminer and the same think happened

**Comment by pdieppa** [comment]
Another thing to check are the OC settings of the card. I don't have a 3080ti so I'm unfamiliar with what works, but I would set afterburner back to default and only limit the power and then see if it can mine successfully at stock settings.

**Comment by Andr65cmd** [comment]
unfortunately i did and it didnt work, i really dont know what else to try i ve been trying for a week

**Comment by pdieppa** [comment]
Did you try running the card under a different operating system?

**Comment by Andr65cmd** [comment]
not yet i will on hiveos (intresting update the error is coming to whichever gpu i am connecting to the riser)

**Comment by pdieppa** [comment]
Ohhh.... bad riser?

**Comment by Andr65cmd** [comment]
actually no i changed the risers 4 times and i have these ubit ones from amazon.I am thinking something with the psu or motherboard.Any idea what to settings to change on bios ?

**Comment by pdieppa** [comment]
I'm not really sure,  there is an Above 4G memory setting that should be enabled. The other settings I'm used to are related to USB drives and booting from them.

**Comment by Andr65cmd** [comment]
ok, what motherboard are you using ?

**Comment by pdieppa** [comment]
I use a few different ones, 
H110 Pro BTC+ ASRock
TUF GAMING B460M-PLUS AC ASUSTeK
B450M DS3H WIFI V2-CF Gigabyte

**Comment by Andr65cmd** [comment]
cool i have a b550 aorus pro v2 do you think it might be the MB?.Also what psus are you using ?
